crates/sncast/src/main.rs

```diff
@@ -76,7 +76,7 @@ struct Cli {
     account: Option<String>,
 
     /// Path to the file holding accounts info
-    #[clap(short = 'f', long = "accounts-file", global = true)]
+    #[clap(long = "accounts-file", global = true)]
```

Comment:
note: imo it's better to get rid of short option if we can't use `f` or `a` (they are take by other flags) than to use something unrelated letter.